File: back/src/main/java/com/example/back/controllers/dto/UserBetChoiceDto.java
package com.example.back.controllers.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class UserBetChoiceDto {
    private Long user1Id;
    private Long user2Id;
    private int user1Choice;
    private int user2Choice;
}

File: front/amadeusSports/src/app/dto/UserBetChoiceDto.ts
export class UserBetChoiceDto {
    user1Id!: number;
    user2Id!: number;
    user1Choice!: number;
    user2Choice!: number;
    constructor(user1Id: number,
        user2Id: number,
        user1Choice: number,
        user2Choice: number) {
        this.user1Id = user1Id;
        this.user2Id = user2Id;
        this.user2Choice = user2Choice;
        this.user1Choice = user1Choice;
    }
}